Sources/Moya/MoyaProvider.swift

```diff
@@ -25,8 +25,15 @@ public struct ProgressResponse {
     }
 }
 
+public protocol MoyaProviderProtocol: class {
```

Comment:
I don't mind either so I did a quick research what we use in the project: we use `...Type` & `...able` for protocols and I agree, I'll change to `MoyaProviderType` 👍